docs/api.md

```diff
@@ -1507,18 +1505,18 @@ There are no standard failure modes for this endpoint.
 * `POST /get_random_bytes`
 * `POST /account/create`
 * `POST /account/login?keys=true`
+* `POST /account/device`
```

Comment:
While I was updating the docs, I notice the position of the device registration request in these flows didn't match reality. Figured it was okay to fix while I was in here.